src/server/routes/schedules.js

```diff
@@ -0,0 +1,12 @@
+const router = require('express').Router();
+const {
+  scheduleController: { get, getOne, post, patch, remove },
+} = require('../controllers');
```

Comment:
nice use of destructuring in the routes. I'll make sure to do this in the future for sure.